test/test_main.py

```diff
@@ -33,11 +33,14 @@ def test_command_help():
 
 
 def test_command_lib_list():
+    """
+    When ouput is empty, nothing is printed out, no matter the output format
+    """
     result = run_command('lib list')
     assert result.ok
     assert result.stderr == ''
```

Comment:
also change this to 

`assert '' == result.stderr`

to keep the style consistent